data/human/news.txt

```diff
@@ -1990,6 +2093,75 @@ news "conspiracy theorist"
 
 
 
+news "player's crew member"
+	location
+		government "Republic" "Free Worlds" "Neutral" "Independent" "Syndicate"
+	to show
+		"flagship crew" > 30
```

Comment:
Changing to 40. If we go beyond that, then the player will almost never encounter this (since there are not many ships that require such large amounts of crew).

```suggestion
		"flagship crew" > 40
```